timescaledb/tutorials/grafana/visualizations/time-series.md

```diff
@@ -0,0 +1,194 @@
+# Build a time-series graph in Grafana 
+A time-series graph is a line graph consisting series of data points in order on
+cartesian coordinates. The X axis represents time and the Y axis represents the 
+value of the data point. They are useful for visualizing trends and fluctuations 
+in the data. The time-series graph is the most common type of graph in Grafana
+and is therefore the default panel type.
+
+They answer questions like:
+
+* What is the hourly stock price of AMD today?
+* How many users visited a website page each day in the past week?
+* What was the temperature in yesterday?
+
+## Data for Grafana time-series graphs 
+To plot a time-series graph, Grafana requires you to provide a time column and 
+the respective value column. To plot multiple time-series graphs in a single 
+panel you need to provide multiple value columns.
+
+This is an example of valid time-series data:
+```bash
+Time                | Value 1 | Value 2 |
```

Comment:
```suggestion
Time                | Value_1 | Value_2 |
```
Really minor, but we generally try to avoid spaces in column names per convention. I realize this is just an example, but consistency is good.